onnx_tf/common/__init__.py

```diff
@@ -98,3 +100,51 @@ def get_output_node_names(graph_def):
     nodes[node.name] = node
     input_names.update(set(node.input))
   return list(set(nodes) - input_names)
+
+
+class Deprecated:
+  """Add deprecated message when function is called.
+
+  Usage:
+    from onnx_tf.common import deprecated
+
+    @deprecated
+    def func():
+      pass
+
+    UserWarning: func is deprecated. It will be removed in future release.
+
+    @deprecated({"arg": "Message", "arg_1": deprecated.REMOVE})
+    def func(arg, arg_1, arg_2):
+      pass
+
+    UserWarning: arg of func is deprecated. Message
```

Comment:
So in message, we can write something like "Use [another op] instead"?
Sounds like a useful feature we should make use of.